### DISCUSSION TITLE
how do i make the window move around in ke 1.5.1?

### DISCUSSION THREAD

**Discussion by hifishlol** [KadeArchive/Kade-Engine#2690] [Q&A]
how do i make the window move around like vs flexy?

**Comment by isakube** [comment]
brb making a Gist for this

**Reply by hifishlol** [reply]
huh so i wonder what happened to this

**Reply by isakube** [reply]
i do too sometimes

**Comment by ghost** [comment]
FlxTween.tween(openfl.Lib.application.window, { y: 30, x: 50 }, 0.2, { type: FlxTween.PINGPONG, ease: FlxEase.quadInOut });
